index.md

```diff
@@ -4,3 +4,11 @@ layout: default
 
 {% include 01-name.md %}
 
+<br>
+
+{% include 02-image.md %}
```

Comment:
![Image of Coolzerg](https://s.yimg.com/rz/p/yahoo_frontpage_en-US_s_f_p_205x58_frontpage_2x.png)